static/image/refresh-disabled.svg

```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24">
```

Comment:
You can run `svgo` to reduce the size of this file, see https://github.com/svg/svgo